distribution/docs/src/main/jdocs/templates/documentation.ftl

```diff
@@ -67,4 +67,79 @@ include::${project.build.directory}/doc-contents/_contents/config.adoc[]
 [appendix]
 == Application References
 
-<#include "application-reference.ftl">
\ No newline at end of file
+<#include "application-reference.ftl">
+
+// Include iframeResizer to fix scolling issue when displayed in admin-ui.
+++++
+<script type="text/javascript" src="/admin/iframe-resizer/2.6.2/js/iframeResizer.contentWindow.min.js"></script>
+<script type="text/javascript" >
+"use strict";
+
+function makeTocExpandable(level) {
+	const tocHeading = 	document.querySelectorAll(".sectlevel" + level + ">li");
+
+	for (var i = tocHeading.length - 1; i >= 0; i--) {
+		const childLevels = tocHeading[i].querySelector("ul li");
+		if (childLevels) {
+			tocHeading[i].innerHTML ='<a class="expandable">+ </a>' + tocHeading[i].innerHTML + '';
+		}
+	}
+	const levelUp = level + 1
+	const higherlevels = document.querySelectorAll(".sectlevel" + levelUp );
+
+	for (var i = higherlevels.length - 1; i >= 0; i--) {
+		higherlevels[i].classList.add("collapsed");
+	}
+
+}
+
+function expandList(section) {
+	const currentSection = section.target.parentNode;
+
+	currentSection.querySelector("ul").classList.remove("collapsed");
+	currentSection.querySelector(".expandable").innerHTML = "- ";
+	currentSection.querySelector(".expandable").removeEventListener("click", expandList);
+	currentSection.querySelector(".expandable").classList.add("expanded");
+	currentSection.querySelector(".expanded").classList.remove("expandable");
+	currentSection.querySelector(".expanded").addEventListener("click", collapseList);
+}
+
+function collapseList(section) {
+	const currentSection = section.target.parentNode;
+
+	currentSection.querySelector("ul").classList.add("collapsed");
+	currentSection.querySelector(".expanded").innerHTML = "+ ";
+	currentSection.querySelector(".expanded").removeEventListener("click", collapseList);
+	currentSection.querySelector(".expanded").classList.add("expandable");
+	currentSection.querySelector(".expandable").classList.remove("expanded");
+	currentSection.querySelector(".expandable").addEventListener("click", expandList);
+}
+
+function addTocListeners() {
+	const expandableSections = document.querySelectorAll(".expandable")
+	const expandedSections = document.querySelectorAll(".expanded")
+
+	for (var i = expandableSections.length - 1; i >= 0; i--) {
+		expandableSections[i].addEventListener("click", expandList);
+	}
+	for (var i = expandedSections.length - 1; i >= 0; i--) {
+		expandedSections[i].addEventListener("click", collapseList);
+	}
+}
+
+document.onreadystatechange = () => {
+	if (document.readyState === 'complete') {
+	  // The page is fully loaded
+		makeTocExpandable(1);
```

Comment:
Couldn't we just do a makeTocExpandable(5) here and then in makeTocExpandable just loop from 1 to level?